src/server/routes/defaultRoute.js

```diff
@@ -28,6 +28,7 @@ const assets = require(process.env.RAZZLE_ASSETS_MANIFEST); //eslint-disable-lin
 const getAssets = () => ({
   css: assets.client.css ? assets.client.css : undefined,
   js: [assets.client.js],
+  mathJaxConfig: { js: '../../mathjax/config.js' },
```

Comment:
You need to access the assets to object to get the correct file path. This won't work when NODE_ENV === production